packages/salesforcedx-sobjects/test/integration/sobjectDescribeTest.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  CliCommandExecutor,
+  CommandOutput,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import { expect } from 'chai';
+import * as jsforce from 'jsforce';
+import { SObjectDescribe } from '../../src/describe/sObjectDescribe';
+
+describe('Fetch sObjects', function() {
+  // tslint:disable-next-line:no-invalid-this
+  this.timeout(10000);
+  let username: string;
+  const sobjectdescribe = new SObjectDescribe();
+  const scratchDefFilePath =
+    '/test/integration/config/project-scratch-def.json';
+  before(async function() {
+    const execution = new CliCommandExecutor(
+      new SfdxCommandBuilder()
+        .withArg('force:org:create')
```

Comment:
How does this test pass locally? I think this can pass on the autobuilds because we created a default deb hub. See https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/.appveyor.yml#L29 and https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/.travis.yml#L57

But how do we run this test locally?

Reply:
I have a default dev hub enabled locally. I think we should make that a requirement since it's hard to automate the authorization of a new dev hub.

Reply:
Correct. In that case, do you want to add it to the doc/testing.md? Since if your changes go in, it would no longer possible to just use `npm run test` if you don't have a dev hub set up.